webapp/channels/src/components/emoji/add_emoji/add_emoji.tsx

```diff
@@ -384,18 +389,14 @@ export default class AddEmoji extends React.PureComponent<AddEmojiProps, AddEmoj
                                 />
                             </Link>
                             <SpinnerButton
-                                data-testid='save-button'
-                                className='btn btn-primary'
-                                type='submit'
+                                testId='save-button'
+                                emphasis='primary'
```

Comment:
0/5 but perhaps `emphasis` should default to `primary`

Reply:
I probably left the emphasis as required at the beginning to make sure I assigned the right one on every component, but yes, you are right.